custom-namespaces.yaml

```diff
@@ -1131,14 +1131,23 @@ subscription_platform:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.subscription_platform_derived.apple_subscriptions_v1
+      measures:
+        count:
+          type: count
```

Comment:
Defining measures like this is a new feature for table views just added in #686.